client/image_tag_test.go

```diff
@@ -30,11 +30,35 @@ func TestImageTagInvalidReference(t *testing.T) {
 	}
 
 	err := client.ImageTag(context.Background(), "image_id", "aa/asdf$$^/aa")
-	if err == nil || err.Error() != `Error parsing reference: "aa/asdf$$^/aa" is not a valid repository/tag` {
+	if err == nil || err.Error() != `Error parsing reference: "aa/asdf$$^/aa" is not a valid repository/tag: invalid reference format` {
 		t.Fatalf("expected ErrReferenceInvalidFormat, got %v", err)
 	}
 }
 
+func TestImageTagInvalidSourceImageName(t *testing.T) {
+
```

Comment:
can you remove this blank line? overall `Go` coding style is to not add whitespace where not needed